pkg/sql/codegen/couler/katib.go

```diff
@@ -0,0 +1,98 @@
+// Copyright 2019 The SQLFlow Authors. All rights reserved.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package couler
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"sqlflow.org/sqlflow/pkg/sql/codegen/attribute"
+	"sqlflow.org/sqlflow/pkg/sql/ir"
+)
+
+var defaultKatibDockerImage = "sqlflow/sqlflow"
```

Comment:
Can resue `defaultDockerImage` ?
https://github.com/sql-machine-learning/sqlflow/blob/4646803832cac20b303774eea989182a623520cf/pkg/sql/codegen/couler/codegen.go#L26-L28

And I didn't found where to use it in this PR.

Reply:
just to maintain parity as the other yaml was having that